scripts/build.ps1

```diff
@@ -175,9 +178,20 @@ function CMake-Generate {
     if ($DynamicCRT) {
         $Arguments += " -DQUIC_STATIC_LINK_CRT=off"
     }
+    if ($PGO) {
+        $Arguments += " -DQUIC_PGO=on"
+    }
     $Arguments += " ../../.."
 
     CMake-Execute $Arguments
+
+    if ($PGO -and $Config -eq "Release") {
+        # Manually edit project file, since CMake doesn't seem to have a way to do it.
+        $FindText = "  <PropertyGroup Label=`"UserMacros`" />"
+        $ReplaceText = "  <PropertyGroup Label=`"UserMacros`" />`r`n  <PropertyGroup><LibraryPath>`$(LibraryPath);`$(VC_LibraryPath_VC_$($Arch)_Desktop)</LibraryPath></PropertyGroup>"
+        $ProjectFile = Join-Path $BuildDir "src\bin\msquic.vcxproj"
+        (Get-Content $ProjectFile) -replace $FindText, $ReplaceText | Out-File $ProjectFile
```

Comment:
The correct way to do this would be to edit the abstract XML tree, not the raw string data. Is that a challenge here?

Reply:
This file is dynamically generated by cmake and is very large. I don't think it would be a good idea to try to interact with it as XML. I agree that'd a nicer way, from a design/arch perspective, but is not practical in this case. If you really disagree and want to try to do it, feel free to give it a try.